.eslintrc.json

```diff
@@ -1,16 +1,28 @@
 {
     "plugins": [
+//        "@angular-eslint/eslint-plugin",
         "@typescript-eslint",
         "@typescript-eslint/tslint",
         "chai-friendly"
     ],
     "extends": [
+//        "eslint:recommended",
+//        "plugin:@typescript-eslint/recommended",
+//        "plugin:@typescript-eslint/recommended-requiring-type-checking",
+//        "plugin:@angular-eslint/recommended",
         "plugin:prettier/recommended",
         "prettier"
+//        "eslint-config-prettier"
```

Comment:
There are many commented out lines in this file. Are they left there for a reason or were just forgotten?

Reply:
@krusche should we leave it, should I delete them?